src/components/ObsDetails/ObsDetailsContainer.js

```diff
@@ -141,34 +140,17 @@ const ObsDetailsContainer = ( ): Node => {
 
   const localObservation = useLocalObservation( uuid );
 
-  const fetchRemoteObservationQueryKey = useMemo(
-    ( ) => ( ["fetchRemoteObservation", uuid] ),
-    [uuid]
-  );
   const fetchRemoteObservationEnabled = (
     !remoteObsWasDeleted
-    && !!isOnline
-    && localObservation?.wasSynced( )
+    && ( !localObservation || localObservation?.wasSynced( ) )
```

Comment:
When opening ObsDetails of someone else's observation it did not fetch from the server. This is weird because I think I only refactored this code into the useRemoteObservation hook without changing the desired behavior. If I make a check here for not having a local observation it does fetch from the server and the observation loads in the UI. Please, let me know if that might break other things.

Reply:
Yeah, the logic in `main` is wrong, and it is only working by coincidence because when useQuery received an `enabled` value of `undefined`, it treats that as `true` and runs the query. This is apparently [desired behavior](https://github.com/TanStack/query/issues/2269), even though it's somewhat insane.

As far as I can tell, your changes both fix what was probably my logical problem in main, as well as prevent similar problems by ensuring the `enabled` value is always boolean.